src/core/scene/a-scene.js

```diff
@@ -45,13 +45,14 @@ module.exports = registerElement('a-scene', {
 
     createdCallback: {
       value: function () {
-        this.isMobile = isMobile;
+        this.init();
         this.isIOS = isIOS;
+        this.isMobile = isMobile;
         this.isScene = true;
         this.object3D = new THREE.Scene();
+        this.renderBinded = this.render.bind(this);
         this.systems = {};
         this.time = 0;
-        this.init();
```

Comment:
I would kip init at the bottom. There's logic that might rely on the variables set above. For instance the logic that initializes systems might want to check for `isIOS` or `isMobile`